pandas/core/arrays/interval.py

```diff
@@ -39,6 +41,14 @@
 import pandas.core.common as com
 from pandas.core.indexes.base import ensure_index
 
+try:
+    import pyarrow
+
+    _PYARROW_INSTALLED = True
+except ImportError:
```

Comment:
can u make this into a function and put in common location

Reply:
I moved this for now into an `_arrow_utils.py` file in the `arrays` directory, we can then put some common functions in that file as well